pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -149,8 +149,8 @@ func (c *contract) Reveal(ctx context.Context, storageDepth uint8, reserveCommit
 	request := &transaction.TxRequest{
 		To:          &c.incentivesContractAddress,
 		Data:        callData,
-		GasPrice:    sctx.GetGasPrice(ctx),
-		GasLimit:    sctx.GetGasLimitWithDefault(ctx, 3_000_000),
+		GasPrice:    new(big.Int).Div(new(big.Int).Mul(sctx.GetGasPrice(ctx), big.NewInt(15)), big.NewInt(10)),
```

Comment:
So what is this calculation doing? Can we add some comments?

Reply:
Actually, what happens if sctx.GetGasPrice return nil?  If the context doesn't have a gas-price key, that's what will be coming back.   And I failed to track down where ctx actually comes from, but I'm thinking this multiplicative factor is actually in the wrong place, but needs to be gasPrice has been resolved at https://github.com/ethersphere/bee/blob/00aada8a51a06e2997ba7a4f1df6b717c853d99e/pkg/transaction/transaction.go#L276